host/sgx/sgxquoteprovider.h

```diff
@@ -24,14 +24,23 @@ typedef struct _oe_sgx_quote_provider
     sgx_ql_free_revocation_info_t free_revocation_info;
     sgx_get_qe_identity_info_t get_qe_identity_info;
     sgx_free_qe_identity_info_t free_qe_identity_info;
```

Comment:
Shouldn't these be replaced by the new functions?

Reply:
Thanks, replaced.